### PR TITLE
Refactor rails runner script

### PR DESCRIPTION
**Pull request by jsugarman**

### CODE REVIEW

File: lib/geckoboard_publisher/report.rb [hunk 1]
```diff
@@ -82,5 +84,21 @@ def test_client
...
+    def cron_log string
+      puts "#{DateTime.current}: #{string}" unless Rails.env.test?
+    end
+    # rubocop:enable Rails/Output
+
+  end
+end
+
+class Object
+  def to_sb
```
Comment (colinbruce) [inline lib/geckoboard_publisher/report.rb]: As a matter of interest... why `to_sb`?  What does the sb stand for?

File: lib/geckoboard_publisher/report.rb [hunk 2]
```diff
@@ -82,5 +84,21 @@ def test_client
...
+    return 'FAILURE' if [FalseClass, NilClass].include?(self.class)
+    return 'success' if self.class == TrueClass
```
Comment (colinbruce) [inline lib/geckoboard_publisher/report.rb]: I'd match the case either both UPPER or both Capitalize

File: run.sh [hunk 3]
```diff
@@ -11,30 +11,32 @@ esac
...
 
-    echo "installing and running cron"
-    apt-get update
-    apt-get install -y cron
-    cron
+    echo "installing cron"
+    apt-get update && apt-get install -y cron
+
+    echo "starting cron"
+    service cron start
```
Comment (colinbruce) [inline run.sh]: Filippos recommended `service cron restart`

File: lib/geckoboard_publisher/report.rb (hunk 1 above)
Comment (jsugarman) [inline lib/geckoboard_publisher/report.rb]: good question, to_string_boolean would be more descriptive - what do you think?

File: lib/geckoboard_publisher/report.rb (hunk 2 above)
Comment (jsugarman) [inline lib/geckoboard_publisher/report.rb]: fair enough - was wanting to emphasise failures in the log - old habits

File: run.sh (hunk 3 above)
Comment (jsugarman) [inline run.sh]: yeah i was thinking to do that. have you tested it. I assume restart will start if currently stopped?!

**Pull request merged**